lib/src/flutter.dart

```diff
@@ -0,0 +1,67 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:io/ansi.dart' as ansi;
+import 'package:path/path.dart' as p;
+
+import 'options.dart';
+import 'utils.dart';
+
+Future<void> runFlutterBuild(
+  String pkgDirectory,
+  Map<String, String> targets,
+  Options options,
+) async {
+  if (targets.entries.length != 1) {
+    // TODO(johnpryan): investigate how the flutter command handles targets
+    print('Warning: only 1 target (web) is supported for Flutter apps');
+    return;
```

Comment:
We should have a throw or something here, right? We want a non-zero exit, etc...

Maybe a `PeanutException`.

Reply:
👍